lib/renderer/webview/webview.h

```diff
@@ -2298,8 +2308,22 @@ struct webview_priv
                     }];
       if ([NSApp runModalForWindow:panel] == NSModalResponseOK)
       {
-        const char *filename = [[[panel URL] path] UTF8String];
-        strlcpy(result, filename, resultsz);
+        /* https://stackoverflow.com/a/15318065/6748719 */
+        int filesCount = [[panel URLs] count];
+        char **filePaths = (char **)malloc(sizeof(char *) * (filesCount + 1));
+        int i;
+        for (id url in [panel URLs]) {
+          const char *filePath = [[url path] UTF8String];
+          int len = strlen(filePath);
+          char *filePathCopy = (char *)malloc(sizeof(char) * (len + 1));
```

Comment:
I think this is redundant. `[[url path] UTF8String]` allocates the extra memory so you could just store those directly into filepaths. We also need to free this memory too 😄